net-libs/libvncserver/libvncserver-0.9.12-r1.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="LibVNCServer"
```

Comment:
In both cases you use this as `${MY_PN}-${PV}`. Why not just make it into `MY_P`?